### PR TITLE
new package: py-lmdb

### PR DESCRIPTION
**Pull request by aweits**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-lmdb/package.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+class PyLmdb(PythonPackage):
+    """ Universal Python binding for the LMDB 'Lightning' Database"""
+
+    pypi = "lmdb/lmdb-1.3.0.tar.gz"
+    homepage = "https://github.com/jnwatson/py-lmdb/"
+
+    version('1.3.0', sha256='60a11efc21aaf009d06518996360eed346f6000bfc9de05114374230879f992e')
+
+    depends_on('python@3.4:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-lmdb/package.py]:
```suggestion
    depends_on('python@2.7:2,3.4:', type=('build', 'run'))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-lmdb/package.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-lmdb/package.py:1]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-lmdb/package.py:1]: We should probably just do a mass-update for all packages... I certainly wouldn't require this change until the `spack create` template has been updated.

**Review by adamjstewart — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-lmdb/package.py (hunk 2 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-lmdb/package.py]: Wait, was it already changed? When?

**Comment by adamjstewart** [comment]
Can you undo the license change? It's causing the tests to fail

**Review by adamjstewart — APPROVED** [review]

**Comment by tldahlgren** [comment]
> Can you undo the license change? It's causing the tests to fail

Apologies.  I didn't realize the copyright year was being checked when I made the suggestion.

**Pull request merged**